reflect/src/main/java/dagger/reflect/BindingMap.java

```diff
@@ -98,6 +99,23 @@ Builder addIntoSet(Key key, Binding binding) {
       return this;
     }
 
+    Builder addIntoMap(Key key, Object mapKey, Binding mapValueBinding) {
```

Comment:
Maybe using 'entry' would be a good prefix. Similar to 'element' for sets.

Reply:
`bindingKey`?

Reply:
I could also just write some docs...

Reply:
Settled on entryKey and entryValueBinding as least confusing... for now.